### ISSUE TITLE
Track performance of pretraining models on TARGET validation data to better visualize transfer training curve

### ISSUE THREAD

**Issue by zspald** [coganlab/RNN_phoneme_decoding_micro#86]
Don't think this will work with teacher-forcing validation via tf model.fit(), BUT I can define the full model for the target pt by replacing the convolutional input to the original encoder-decoder, then assign that and the associated inference models as the models used by the seq2seq validation loss and accuracy callback

**Comment by zspald** [comment]
Would need a way to have target convolutional layer trained for this to work. Otherwise RNN weights could transfer to target network used for seq2seq validation but the target convolutional layer would extract nonsense
